installation/local/roles/web_install/templates/090-cloudsnitch.conf.j2

```diff
@@ -11,6 +11,7 @@
 	# DocumentRoot /var/www/html
 
 	SSLEngine on
+    SSLProtocol {{ cloud_snitch_web_ssl_protocol }}
```

Comment:
```suggestion
      SSLProtocol {{ cloud_snitch_web_ssl_protocol }}
```

Reply:
Went ahead and changed all tabs to spaces in the vhost conf.